live-examples/html-examples/global-attributes/tabindex.html

```diff
@@ -0,0 +1,5 @@
+<form>
+    <label>First in tab order: <input type="text"></label>
+    <div tabindex="0">Tabbable due to tabindex.</div>
+    <label>Third in tab order: <input type="text"></label>
+</form>
```

Comment:
I found this one a little hard to use, because you have to know to focus the output pane before tabbing.

I am not generally that keen on including instructions in the example, but I think it might help here. I also think it would help to have a div that doesn't use tabindex and is therefore not tabbable. For example:

```
<p class="instructions">Click anywhere in this pane, then try tabbing through the elements.</p>

<form>
    <label>First in tab order: <input type="text"></label>
    <div tabindex="0">Tabbable due to tabindex.</div>
    <div>Not tabbable: no tabindex.</div>
    <label>Third in tab order: <input type="text"></label>
</form>
```

...with possible styling of the instructions:

```
.instructions {
    font-style: italic;
}
```

WDYT?